container/cli.py

```diff
@@ -264,14 +260,19 @@ def __call__(self):
             LOGGING['loggers']['container']['level'] = 'DEBUG'
         config.dictConfig(LOGGING)
 
+        vargs = vars(args)
+        if not vargs['project_name']:
```

Comment:
Outside of `init`, wouldn't this make more sense to be part of the `container.yml` settings? So it doesn't have to be specified every single time the user invokes `ansible-container`?